.github/workflows/apero-docs-preview.yaml

```diff
@@ -0,0 +1,37 @@
+on:
+  schedule:
+    # * is a special character in YAML so we have to quote this string
+    # 3am Pacific = 11am UTC
+    # https://crontab.guru is your friend
+    - cron:  '0 12 * * *'
+
+name: theme-preview
+
+jobs:
+  theme-preview:
+    runs-on: macOS-latest
+    if: github.event_name == 'schedule' || contains(github.event.head_commit.message, '[theme]')
```

Comment:
> contains(github.event.head_commit.message, '[theme]')

Sorry I missed that. You may need to add later the on push event if you want this to really checked.